package.json

```diff
@@ -16,7 +16,8 @@
     "dx:down": "docker compose -f docker/compose-services.yml down",
     "prisma:migrate-dev": "npm run with:env -- npm run prisma:migrate-dev -w @documenso/prisma",
     "prisma:migrate-deploy": "npm run with:env -- npm run prisma:migrate-deploy -w @documenso/prisma",
-    "with:env": "dotenv -e .env -e .env.local --"
+    "with:env": "dotenv -e .env -e .env.local --",
+    "precommit": "npm install && git add package.json package-lock.json"
```

Comment:
Iirc lint staged will happily modify staged files, is this just because package-lock might not be staged?

Reply:
yeah after running the script package-lock.json didn't get committed that's why tryed with script !